test/test426/test-source-map-spec.mjs

```diff
@@ -0,0 +1,59 @@
+import '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+
+import assert from 'node:assert';
+import test from 'node:test';
+import { SourceMap } from 'node:module';
+import { readFileSync } from 'node:fs';
+import path from 'node:path';
+
+const specJson = fixtures.readSync('test426/source-map-spec-tests.json', 'utf8');
+const spec = JSON.parse(specJson);
+
+const kStatus = JSON.parse(readFileSync(path.join(import.meta.dirname, 'status/source-map-spec-tests.json'), 'utf8'));
```

Comment:
Let's make backporting easier
```suggestion
const kStatus = JSON.parse(readFileSync(new URL('./status/source-map-spec-tests.json', import.meta.url), 'utf8'));
```

Reply:
`import.meta.dirname` is a semver-minor change, and it's already on all the LTS branches. Would you mind elobrating on how would this make backporting harder?

Reply:
I don't think it's on v18.x (nor earlier): https://github.com/nodejs/node/blob/b6df12819da0fd1e1a8dbeb65f5ddc54cb267ddf/doc/api/esm.md#L348-L354